api/models/User.js

```diff
@@ -1,21 +1,97 @@
-const mongoose = require('mongoose');
-const Schema = mongoose.Schema;
-
- const userSchema = new Schema({
-     name: {
-         type: String,
-         required: true
-     },
-     email: {
-         type: String,
-         required: true
-     },
-     password: {
-         type: String,
-         required: true
-     }
- },{timestamps: true})
-
- const User = mongoose.model('User', userSchema)
-
- module.exports = User;
\ No newline at end of file
+var mongoose = require('mongoose');
+const jwt = require('jsonwebtoken');
+const bcrypt = require('bcrypt');
+const confiq = require('../config/config').get(process.env.NODE_ENV);
+const salt = 10;
+
+const userSchema = mongoose.Schema({
+    name: {
+        type: String,
+        required: true,
+        unique: true,
```

Comment:
I am not sure if the name field should be unique.... Two "John Doe" is not a problem, the field that is unique generally is email.

Reply:
I agree shouldn't be unique

Reply:
Solved!